config.py

```diff
@@ -1,7 +1,17 @@
 import logging
 import os
 
-COBOT_ROOT = os.environ.get('COBOT_ROOT', os.getcwd())
+BOT_ROOT_KEY = 'BOT_ROOT'
+
+# Deprecate COBOT_ROOT
+if 'COBOT_ROOT' in os.environ and BOT_ROOT_KEY not in os.environ:
+    logging.warning(
+        "Environment variable {} is deprecated, use {} instead."
+        .format('COBOT_ROOT', BOT_ROOT_KEY))
+    # Add support for COBOT_ROOT
```

Comment:
Add support for COBOT_ROOT?

Reply:
Sorry I should have been more precise, I mean the Python comment itself. I see that it's deprecated when looking at the message above, this Python comment just confuses me^^ I directly asked myself "What additional support is being offered here besides handling the deprecation?", as `COBOT_ROOT` was already "supported" prior to this change.